test/config/config.yaml

```diff
@@ -6,7 +6,7 @@ prometheusEndpoint: /metrics
 prometheusAddr: :2112
 grpcMessageSize: 100000000
 network: regtest
-queueURL: nats://nats:4222
+queueURL: nats://nats-1:4222,nats://nats-2:4223,nats://nats-3:4224
```

Comment:
The PR is looking great 👍 

I just have one question, why do we need 3 instances of NATS? Is it to make sure no messages are lost in case one NATS instance failing/disconnecting? 

Even if so, it should be extremely rare for that to happen, so maybe it's not worth the resources and time for e2e tests to spin up 3 instances? One should be enough? WDYT?

Reply:
That is right. Just one would be enough for sure.
I just thought it is good to have a cluster here in the e2e test because that also allows to easily lookup how a cluster needs to configured for anybody who wants to run ARC with a JetStream-enabled NATS cluster. So It serves a kind of documentation purposes. What do you think?

Reply:
I think it was a good idea to use a cluster, but @kuba-4chain has a point. Maybe we should limit instances to 2 for easier and faster running tests on local machines.

Maybe we could also apply the same approach to other services?

Reply:
Ok, I'll reduce it to 2 instances.

@arkadiuszos4chain Which other services do you mean?

Reply:
Nodes and blocktx, I think we are running 3 nodes for tests and scale blocktx to 7 instances

Reply:
@arkadiuszos4chain Sounds good. But let's do it in a separate PR